sky/utils/command_runner.py

```diff
@@ -431,10 +432,13 @@ def run(
             cmd,
             process_stream,
             separate_stderr,
-            # A hack to remove the following bash warnings (twice):
+            # A hack to remove the following SSH warning+bash warnings (twice):
+            # Warning: Permanently added 'xx.xx.xx.xx' to the list of known
```

Comment:
```suggestion
            #  Warning: Permanently added 'xx.xx.xx.xx' to the list of known
```
add a space to align with the `bash: cannot set terminal process group` below?